scripts/pxc-keyring-test-pks.sh

```diff
@@ -0,0 +1,2750 @@
+############################################################
+# Created by Mohit Joshi                                   #
+# Creation date: 20-Jan-2022                               #
+############################################################
+#!/bin/bash
+set +xe
+
+# Same for the same os
+
+BASEDIR=/usr/
+
+FILE_PLUGIN=0
+VAULT_PLUGIN=0
+
+PXC_START_TIMEOUT=600
+
+
+SSH_EOF=$(ssh root@DB1_PUB """cat /etc/os-release""")
+echo $SSH_EOF
+
+
+if [ $(echo $SSH_EOF  | grep debian | wc -l) -eq 1 ]
```

Comment:
instead of sending the output to 3 pipes, we can do it simply using
if [[ "$SSH_EOF" =~ "debian" ]]; then